test/controllers/confirm.company.controller.unit.ts

```diff
@@ -162,6 +162,33 @@ describe("Confirm company controller tests", () => {
     expect(response.status).toEqual(200);
     expect(mockCreateConfirmationStatement).not.toHaveBeenCalled();
     expect(response.text).toContain("You cannot use this service - File a confirmation statement");
+    expect(response.text).toContain("https://www.gov.uk/government/publications/confirmation-statement-cs01");
+  });
+
+  it("Should redirect to use paper stop screen when the eligibility status code is INVALID_COMPANY_TYPE_PAPER_FILING_ONLY, type scottish-partnership", async () => {
+    validCompanyProfile.type  = "scottish-partnership";
```

Comment:
can you set the validCompanyProfile.type back to it's original value at the end of the test so it doesn't affect other tests (the validCompanyProfile will have unexpected type if it is modified by other tests)